src/contracts/core/AllocationManager.sol

```diff
@@ -905,4 +1008,45 @@ contract AllocationManager is
         // less than or equal to
         return status.registered || block.number <= status.slashableUntil;
     }
+
+    /// @inheritdoc IAllocationManager
+    function getRedistributionRecipient(
+        OperatorSet memory operatorSet
+    ) public view returns (address) {
+        // Load the redistribution recipient and return it if set, otherwise return the default burn address.
+        address redistributionRecipient = _redistributionRecipients[operatorSet.key()];
+        return redistributionRecipient == address(0) ? DEFAULT_BURN_ADDRESS : redistributionRecipient;
+    }
+
+    /// @inheritdoc IAllocationManager
+    function isRedistributingOperatorSet(
+        OperatorSet memory operatorSet
+    ) public view returns (bool) {
+        return _redistributionRecipients[operatorSet.key()] != address(0);
+    }
+
+    /// @inheritdoc IAllocationManager
+    function getSlashCount(
+        OperatorSet memory operatorSet
+    ) external view returns (uint256) {
+        return _slashCount[operatorSet.key()];
+    }
+
+    /// @inheritdoc IAllocationManager
+    function isOperatorRedistributable(
+        address operator
+    ) external view returns (bool) {
+        EnumerableSet.Bytes32Set storage allocatedSets = allocatedSets[operator];
+        EnumerableSet.Bytes32Set storage registeredSets = registeredSets[operator];
+
+        uint256 allocatedLength = allocatedSets.length();
+        for (uint256 i = 0; i < allocatedLength; ++i) {
+            bytes32 operatorSetKey = allocatedSets.at(i);
+            // Check if the operator is both allocated and registered to this set
+            if (registeredSets.contains(operatorSetKey) && _redistributionRecipients[operatorSetKey] != address(0)) {
```

Comment:
Bug: see my previous comment. https://github.com/Layr-Labs/eigenlayer-contracts/pull/1355/files#r2105325283

This should call `isRedistributingOperatorSet(operatorSetKey)` rather than checking the `_redistributionRecipient` manually.

Reply:
Actually, I'm wrong - newly-created non-redistributing operator sets have this set to 0.